memgpt/local_llm/grammars/gbnf_grammar_generator.py

```diff
@@ -266,8 +279,14 @@ def generate_gbnf_float_rules(max_digit=None, min_digit=None, max_precision=None
 
 
 def generate_gbnf_rule_for_type(
-    model_name, field_name, field_type, is_optional, processed_models, created_rules, field_info=None
-) -> Tuple[str, list]:
+    model_name,
+    field_name,
+    field_type,
+    is_optional,
+    processed_models,
+    created_rules,
+    field_info=None,
+) -> tuple[str, list[str]]:
```

Comment:
Is there a reason to go from `Tuple` to `tuple`?

Why not `Tuple[str, List[str]]`?

Reply:
No I think some kind of formatter did that. I just copied the updated grammar generator from my llama-cpp-agent framework.